doc/sphinx/language/core/modules.rst

```diff
@@ -1010,3 +1010,75 @@ subdirectories of path). See the command :cmd:`Declare ML Module` in
 
 See :ref:`command-line-options` for a more general view over the Coq command
 line options.
+
+.. _controlling-locality-of-commands:
+
+Controlling the scope of commands with locality attributes
+----------------------------------------------------------
+
+Many commands have effects that apply only within a specific scope,
+typically the section or the module in which the command was
+called. Locality :term:`attributes <attribute>` can alter the scope of
+the effect. Below, we give the semantics of each locality attribute
+but also alert that a few commands give a different meaning to the
+:attr:`local` and :attr:`global` attributes.
+
+.. attr:: local
+
+   The :attr:`local` attribute limits the effect of the command to the
+   current scope (section or module).
+
+   The ``Local`` prefix is an alternative syntax for the :attr:`local`
```

Comment:
```suggestion
   The ``Local`` :n:`@legacy_attr` is an alternative syntax for the :attr:`local`
```

or "legacy attribute" if that can be made to hyperlink

Reply:
I prefer to keep as is.